staging/src/k8s.io/component-base/metrics/testutil/testutil_test.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testutil
+
+import "testing"
+
+func TestNewFakeKubeRegistry(t *testing.T) {
+	if NewFakeKubeRegistry("1.18.0") == nil {
+		t.Fatal("Should get fake registry, but got none.")
+	}
```

Comment:
Do we want more assertions for the fake registry?

Reply:
Not so much necessary. Added a basic test.
Done.